tests/network/localnet/migration_stuntime/libstuntime.py

```
@@ -0,0 +1,149 @@
+"""Helpers for OVN localnet migration stuntime tests."""
+
+from __future__ import annotations
+
+import ipaddress
+import logging
+import re
+from contextlib import contextmanager
+from typing import Final, Generator
```

Ruff (0.15.10)

[warning] Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)